src/_includes/docs/swift-package-manager/migrate-objective-c-plugin.md

```diff
@@ -440,8 +440,8 @@ The example below uses `ios`, replace `ios` with `macos`/`darwin` as applicable.
 
       **Do not commit the migration's changes to your version control system.**
```

Comment:
FYI, I kept this recommendation to not check-in the SPM integration for your example app. This ensures customers using Flutter version lower than 3.24 can continue to build the example app.